src/vivarium_cluster_tools/vipin/perf_report.py

```diff
@@ -189,6 +191,86 @@ def print_stat_report(perf_df: pd.DataFrame, scenario_cols: list):
     logger.info(f"\n{report_df}")
 
 
+def append_perf_data_to_central_logs(perf_df: pd.DataFrame, output_directory: Path) -> None:
+    """Append performance data to the central logs."""
+    perf_df = perf_df.reset_index()
+    # add location data to perf_df
+    artifact_path_col = "scenario_input_data_artifact_path"
+    if artifact_path_col in perf_df.columns:  # if we parallelized across artifact paths
+        perf_df["location"] = perf_df[artifact_path_col].apply(
+            lambda filepath: Path(filepath).stem
+        )
+    else:  # else get from output directory
+        perf_df["location"] = output_directory.parents[3].stem
+
+    ## aggregate scenario information into one column
+    all_scenario_cols = [col for col in perf_df.columns if col.startswith("scenario_")]
+    # remove duplicate scenario information
+    unique_scenario_cols = [
+        col for col in all_scenario_cols if not col.startswith("scenario_run_configuration")
+    ]
+    perf_df["scenario_parameters"] = perf_df[unique_scenario_cols].to_dict(orient="records")
+    perf_df["scenario_parameters"] = perf_df["scenario_parameters"].apply(json.dumps)
+    perf_df = perf_df.drop(all_scenario_cols, axis=1)
+
+    # append child job data
+    NUM_ROWS_PER_FILE = 100_000
+    performance_logs_dir = Path("/mnt/team/simulation_science/pub/performance_logs/")
```

Comment:
I'd add this string to `/psimulate/paths.py`. That's rpobably not quite the correct place for it either but better than a string floating around I think.

Reply:
I didn't like that either and was looking for a file to put it in but I guess I didn't look very closely haha. Sounds good to me